paper/paper.md

```diff
@@ -77,16 +77,11 @@ are not accessible using existing codes.
 When `BurnMan` was first released [@Cottaar:2014], its focus was on
 the seismic properties of the lower mantle, using a single endmember mineral
 database [@Stixrude:2011] as a foundation. Since then,
-its scope has expanded considerably. Some workflows of the current code
-are provided in \autoref{fig:workflows}. 
-
-![Some possible BurnMan workflows. Input/output shown as ellipses, functions as
-rectangles and classes as diamonds.\label{fig:workflows}](figures/workflows.png)
-
+its scope has expanded considerably. 
```

Comment:
This is just a suggestion, but I wondered if it makes sense to remove Figure 1. In the original paper draft on overleaf, you’re using it to explain the individual classes, but without that, I find it a bit confusing, and the workflows that are worth mentioning are already listed in the text.